cluster/images/hyperkube/master.json

```diff
@@ -23,6 +23,7 @@
               "apiserver",
               "--portal-net=10.0.0.1/24",
               "--address=127.0.0.1",
+              "--read-write-port=8080",
```

Comment:
If we can't talk you out of this, it is at least a port on the internal service address, and should always be either 80 or 443.